docs/guide.rst

```diff
@@ -45,7 +44,8 @@ We start by instantiating our :class:`mozanalysis.experiment.Experiment` object:
     exp = Experiment(
         experiment_slug='pref-fingerprinting-protections-retention-study-release-70',
         start_date='2019-10-29',
-        num_dates_enrollment=8
+        num_dates_enrollment=8,
+        app_name="firefox_desktop"
```

Comment:
We'll need to pass the `app_name` into Experiment so we can choose the correct metric based on the platform the experiment is run on